Zotero/Scenes/Detail/PDF/PDFCoordinator.swift

```diff
@@ -334,6 +369,14 @@ extension PDFCoordinator: PdfReaderCoordinatorDelegate {
         controller.popoverPresentationController?.barButtonItem = sender
         self.navigationController?.present(controller, animated: true)
     }
+
+    func showFontSizePicker(sender: UIView, picked: @escaping (UInt) -> Void) {
+        let controller = FontSizePickerViewController(pickAction: picked)
+        controller.preferredContentSize = CGSize(width: 200, height: 400)
+        controller.modalPresentationStyle = UIDevice.current.userInterfaceIdiom == .pad ? .popover : .formSheet
```

Comment:
Sure we can do this. I don't remember why, but we chose this way some time ago and we did it in other popovers as well, so it might be worth creating a ticket to revise all `modalPresentationStyle` instances and switch them all to popovers.

When testing on phone I noticed some usability issues as well, so I'll have to add navigation bar on phone and fix some constraints too.

Reply:
Ok so as discussed personally, there were some issues with popover and we're not using that here.